adapters/reflexstreamer/reflex_test.go

```diff
@@ -9,13 +9,13 @@ import (
 	"github.com/luno/reflex"
 	"github.com/luno/reflex/rpatterns"
 	"github.com/luno/reflex/rsql"
-	"github.com/stretchr/testify/require"
-
 	"github.com/luno/workflow"
 	"github.com/luno/workflow/adapters/adaptertest"
 	"github.com/luno/workflow/adapters/memrecordstore"
 	"github.com/luno/workflow/adapters/memrolescheduler"
 	"github.com/luno/workflow/adapters/memtimeoutstore"
+	"github.com/stretchr/testify/require"
+
```

Comment:
```suggestion
```

Reply:
Going to move this import up above the workflow imports

Reply:
Ah I see this is because all of these are seen as external mods